panorama/client.py

```diff
@@ -68,7 +68,8 @@ async def _get_or_raise(self, path: str, type_: Type[T]) -> T:
 
     async def get_panorama(self, panorama_id: str) -> models.Panorama:
         """Get an individual panorama object by remote id"""
-        return await self._get_or_raise(panorama_id, models.Panorama)
+        path = panorama_id if panorama_id.endswith("/") else panorama_id + "/"
```

Comment:
Approved PR based on this. Are the the test yaml files created automatically?

Reply:
Yep. They're used by the tests to replay HTTP requests made in the code